.github/component_owners.yml

```diff
@@ -29,6 +29,8 @@ components:
     - vishweshbankwar
   src/OpenTelemetry.Extensions.Docker/:
     - iskiselev
+  src/OpenTelemetry.Extensions.Enrichment/:
```

Comment:
Can you add same section for test project?

Reply:
Done. Thank you!